src/main/scala/higherkindness/skeuomorph/openapi/client/print.scala

```diff
@@ -210,8 +284,10 @@ object print {
   }
 
   def defaultTypesAndSchemas[T: Basis[JsonSchemaF, ?]](
-      operationId: OperationId,
+      operationId: Http.OperationId,
       responseOr: Either[Response[T], Reference])(implicit codecs: Printer[Codecs]): (String, String, List[String]) = {
+    def unexpectedErrorName(operationId: Http.OperationId): String =
+      s"${operationId.show}UnexpectedErrorResponse".capitalize
```

Comment:
You can use the `show""` interpolator.  It would fail at compile time when trying to interpolate a value of a type for which there isn't a `Show` instance.  Also, you save the call to `.show`.

```suggestion
          show"$operationId${normalize(name).capitalize}Error".capitalize
```